plugins/catalog-backend/src/service/CatalogBuilder.ts

```diff
@@ -599,16 +599,26 @@ export class CatalogBuilder {
       ...this.placeholderResolvers,
     };
 
-    // These are always there no matter what
+    // The placeholder is always there no matter what
     const processors: CatalogProcessor[] = [
       new PlaceholderProcessor({
         resolvers: placeholderResolvers,
         reader,
         integrations,
       }),
-      new BuiltinKindsEntityProcessor(),
     ];
 
+    // If the user adds a processor named 'BuiltinKindsEntityProcessor',
+    //   skip inclusion of the catalog-backend version.
+    if (
+      !this.processors.find(
+        processor =>
+          processor.getProcessorName() === 'BuiltinKindsEntityProcessor',
+      )
+    ) {
+      processors.push(new BuiltinKindsEntityProcessor());
```

Comment:
Could we always instantiate the `BuiltinKindsEntityProcessor` in a variable, and then compare against its own `getProcessorName()`? It just feels nice to not have to put the constant in here :D Even though we sometimes throw away that instance.

Reply:
Aye, I was trying to avoid throwing away an instance. If you imagine that it won't be an issue, I would agree with that preference as well.

Reply:
Yeah that's OK to just throw it away; it's both cheap and rarely happens in the wild